ui/push-health/GroupedTests.jsx

```diff
@@ -34,6 +48,56 @@ class GroupedTests extends PureComponent {
     });
   };
 
+  addToCheckJob = (job) => {
+    this.setState((prevState) => ({
+      checkedJobs: [...prevState.checkedJobs, job],
+    }));
+  };
+
+  removeFromCheckedJobs = (uncheckedJob) => {
+    const index = this.state.checkedJobs.findIndex((job) => {
```

Comment:
This would be cleaner using `checkedJobs.filter` rather than `findIndex`.  Then take the new array from that function and do `setState({ checkedJobs: filteredCheckedJobs })`